.github/workflows/bridge-ui.yml

```diff
@@ -10,7 +10,7 @@ on:
       - "packages/bridge-ui/**"
 
 jobs:
-  build:
+  build-bridge-ui:
```

Comment:
We have two workflows for bridge UI production, should be consolidated into one

Reply:
Which one should be deleted? Or could you help me merge them together?